azure_functions_worker/dispatcher.py

```diff
@@ -528,7 +537,7 @@ async def _handle__close_shared_memory_resources_request(self, req):
         try:
             for mem_map_name in map_names:
                 try:
-                    success = self._shmem_mgr.free_mem_map(mem_map_name)
+                    success = self._shmem_mgr.free_mem_map(mem_map_name, False)
```

Comment:
For readability, it might be better to put this False in a variable with a name or to do the full param: 
to_delete_backing_resources: False